src/app/components/card/card.component.ts

```diff
@@ -8,23 +8,23 @@ import { TsCardBorderOptions, TsCardComponent } from '@terminus/ui/card';
   templateUrl: './card.component.html',
 })
 export class CardComponent implements AfterViewInit {
-  supportsInteraction = false;
-  centered = false;
-  flat = false;
-  theme = 'primary';
-  border: TsCardBorderOptions = 'none';
+  public supportsInteraction = false;
+  public centered = false;
+  public flat = false;
+  public theme = 'primary';
+  public border: TsCardBorderOptions = 'none';
 
   @ViewChild('transitionCard',  {static: true})
-  transitionCard!: TsCardComponent;
+  public transitionCard!: TsCardComponent;
 
-  ngAfterViewInit() {
+  public ngAfterViewInit() {
     // Default the card to 100px height for demo purposes
     const innerCard = document.getElementById(this.transitionCard.id);
-    innerCard!.style.height = '100px';
+    innerCard.style.height = '100px';
```

Comment:
Interesting... ts doesn't complain about the possible `null` case?

Reply:
I think this may be due to a tsconfig difference.. I think it's because we don't have `"strict": true` in our config here but do in our libraries.